Lesson7/Task3.py

```diff
@@ -0,0 +1,6 @@
+# List comprehension exercise
+# Use a list comprehension to make a list containing tuples (i, j)
+# where `i` goes from 1 to 10 and `j` is corresponding to `i` squared.
+
+comp_list = [(i, i ** 2) for i in range(1, 11) # j = i ** 2
```

Comment:
small typo
```suggestion
comp_list = [(i, i ** 2) for i in range(1, 11)]  # j = i ** 2
```

Reply:
The syntax issue was resolved. @Masergio